source/System.cpp

```diff
@@ -268,6 +268,8 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 			trade[value].SetBase(child.Value(valueIndex + 1));
 		else if(key == "object")
 			LoadObject(child, planets);
+		else if(key == "arrival" && child.Size() >= 2)
```

Comment:
we're guaranteed that the key has a value by line 184
(note that the "trade" key above accesses additional indices, which is why it needs the extra `child.Size()` check)